modmesh/app/cavity2d.py

```diff
@@ -0,0 +1,383 @@
+__package__ = "modmesh.app"
+
+import sys
+import numpy as np
+
+import matplotlib.pyplot as plt
+from matplotlib.backends.qt_compat import QtWidgets
+from matplotlib.backends.backend_qtagg import (
+    FigureCanvas, NavigationToolbar2QT as NavigationToolbar)
+from matplotlib.figure import Figure
+from .. import view
+
+
+def load_app():
+    view.mgr.pycon.writeToHistory("""
+# Use the functions for more examples:
+ctrl.start()  # Start computing. Multiple blank windows will appear, and the
+results will be displayed once the execution is terminated.
+<ctrl> + C # Terminate in terminal
+
+
+ctrl = mm.app.cavity2d.run(interval=10, max_steps=50, profiling=True)
+""")
+    cmd = "ctrl = mm.app.cavity2d.run(interval=10, max_steps=50)"
+    view.mgr.pycon.command = cmd
+
+
+###############################################################################
+N = 81
+dt = 0.001
+tol_p = 1e2
+tol_v = 1e-10
+Re = 100
+dx = 1.0 / (N - 1)
+dy = 1.0 / (N - 1)
+
+P = np.zeros((N + 1, N + 1))
+U = np.zeros((N + 1, N + 1))
+V = np.zeros((N + 1, N + 1))
+U_1 = np.zeros((N + 1, N + 1))
+V_1 = np.zeros((N + 1, N + 1))
+U_2 = np.zeros((N + 1, N + 1))
+V_2 = np.zeros((N + 1, N + 1))
+U_prev = np.zeros((N + 1, N + 1))
+V_prev = np.zeros((N + 1, N + 1))
+P_prev = np.zeros((N + 1, N + 1))
+UC = np.zeros((N, N))
+VC = np.zeros((N, N))
+PC = np.zeros((N, N))
+
+# benchmark
+x_b = [0, 0.0625, 0.0703, 0.0781, 0.0938, 0.1563, 0.2266, 0.2344, 0.5000, 0.8047, 0.8594, 0.9063, 0.9453, 0.9531, 0.9609, 0.9688, 1.0]  # noqa: E501
```

Comment:
You should not use lines that are too long.  Please shorten and remove skipping E501.